vm.go

```diff
@@ -186,6 +187,41 @@ func (vm *VM) evaluateSnippet(filename string, snippet string, kind evalKind) (o
 	return output, nil
 }
 
+func (vm *VM) evaluateDependencies(fileName string, children []ast.Node, dependencies *[]string, stackTrace *[]traceFrame, recursionDepth int) (err error) {
```

Comment:
We should change the name here. Evaluate has a very specific meaning in Jsonnet. It means to take an expression (or a thunk or something) and produce a Jsonnet value according to the rules in https://jsonnet.org/ref/spec.html. 

Maybe `findDependencies`?

Reply:
Changed to `findDependencies`.